src/components/DropList/DropList.ListItem.jsx

```diff
@@ -90,7 +94,7 @@ const ListItem = forwardRef(
         <ListItemTextUI>
```

Comment:
This is perhaps opening a can of worms and I don't think it's a blocker here - but it occurs to me that if we're essentially just providing a list of `<span>s` to Tippy to render in a dropdown, are we creating any accessibility concerns? How does the user know that these are actually more like buttons that can be toggled on and off?

Reply:
all elements have the appropriate aria roles (and aria selected attr too) which downshift takes care of updating, also an element (a11y-status-message) that gets read by a SR in the page gets updated, for example: https://i.hlp.sc/9sVqj9

Reply:
I did have a little out of sync thing in that department that I just fixed... so thanks for bringing this up!